src/cow-react/modules/wallet/web3-react/connection/keystone.tsx

```diff
@@ -0,0 +1,53 @@
+import { ConnectionType } from '@cow/modules/wallet'
+import { getConnectionName } from '@cow/modules/wallet/api/utils/connection'
+
+import { useIsActiveWallet } from 'hooks/useIsActiveWallet'
+import { ConnectWalletOption } from '@cow/modules/wallet/api/pure/ConnectWalletOption'
+import { TryActivation, onError } from '.'
+
+import { initializeConnector } from '@web3-react/core'
+import { MetaMask } from '@web3-react/metamask'
+import { Web3ReactConnection } from '../types'
+
+import { default as KeystoneImage } from '@cow/modules/wallet/api/assets/keystone.svg'
+
+const keystoneCommonOption = {
+  color: '#E8831D',
+  icon: KeystoneImage,
+  id: 'keystone',
+}
+
+export const keystoneInstallOption = {
+  ...keystoneCommonOption,
+  header: 'Keystone',
+  link: 'https://metamask.io/',
```

Comment:
Should we specify a link to keystone website?

Reply:
I don't think so, because we want user to install Metamask actually.

Reply:
Agree is weird, the ideal solution is what Elena mentioned, but no time for that. I'm fine to deploy and see what we get in the feedback